english/light-fire/magic-world/follow/dream/fight/sword/sword.md

```diff
@@ -5,3 +5,11 @@ You try to fight, but you failed fencing in high school.
 You're dead. 
 
 The rat smirks as you sink into the unknown. 
+
+
+
+... But wait! There is a light in the darkness. It seems to be growing in intensity.
+
+[Go into the light](english/marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.